### ISSUE TITLE
[Bug]: When update empty table, the service shutdown

### ISSUE THREAD

**Issue by chrysanthemum-boy** [infiniflow/infinity#530]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Branch name

main

### Commit ID

034a1cb

### Actual behavior

When update empty table, some errors occur and the service shutdown.
![image](https://github.com/infiniflow/infinity/assets/69421435/f4e9ad29-073e-4bcd-bd4f-a24fe6abeee0)

### Steps to reproduce

```Markdown
Just create an empty table, and update the table.
The function test is in /python/test/test_update.py `tes_update_empty_table`
```

**Comment by JinHai-CN** [comment]
Fixed by #577, please test it again.

**Comment by chrysanthemum-boy** [comment]
The bug has already been fixed by #577.
